coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -60,12 +60,31 @@
                       required></v-text-field>
                   </div>
 
-                  <BaseAddress ref="baseAddressNew"
-                    :address="director.deliveryAddress"
-                    :editing="true"
-                    :schema="addressSchema"
-                    @update:address="baseAddressWatcher"
-                  />
+                  <label class="address-sub-header">Delivery Address</label>
+                  <div class="address-wrapper">
+                    <BaseAddress ref="baseAddressNew"
+                       v-bind:editing="true"
+                       :schema="addressSchema"
+                       @update:address="baseAddressWatcher"
+                    />
+                  </div>
+                  <div class="form__row" v-if="entityFilter(EntityTypes.BCorp)">
+                    <v-checkbox
+                      class="inherit-checkbox"
+                      label="Mailing Address same as Delivery Address"
```

Comment:
Note to other reviewers: I confirmed with Scott that this label is correct (even though it's different from the similar label for Office Addresses).